src/RadioButton/RadioButton.component.tsx

```diff
@@ -138,7 +157,22 @@ export class RadioButton extends React.Component<Props> {
     displayStyle: 'stacked',
     name: '',
     theme: Themes.defaultTheme,
-    onChange: () => {},
+    id: uuid.v4(),
+    disabled: false,
+  };
+
+  state = {
+    checked: this.props.defaultChecked,
+  };
+
+  onChange = (event: any) => {
```

Comment:
Console is complaining about this handler, could this be an issue when using the component in external apps?

![image](https://user-images.githubusercontent.com/20873103/57326943-502f3100-7116-11e9-8319-cc3d5fd89241.png)

Reply:
There was confusing with checked and defaultChecked props.

Fixed